### ISSUE TITLE
simclr for audio classification

### ISSUE THREAD

**Issue by FirasHm** [google-research/simclr#23]
Hello, thank you for this great work!
I am using simclr for audio (speech) processing, by extracting features as images from the audio clips and then  use them to train/fine-tune a model. My first experiments provides me 28% (accuracy) for 150 epochs. Here what I did:

1- Extract MFCCs spectrogram from speech audio
2- Fine tune simclr by training the linear classifier

Is it the right way to use simclr ? Do you think that 28% is a "normal" accuracy? I think it is due to the fact that MFCCs images are very similar that the model can not classify them.

Thanks in advance

**Comment by chentingpc** [comment]
I have not applied this to audio so it is hard to know what accuracy to expect, but the approach you described sounds very interesting! I think you may want to double check if the data augmentation still makes sense in the audio images that you generated (e.g. is it too difficult, or is it too easy for the model to learn). Furthermore, if you want to get the best results, you can fine-tuning all layers (e.g by setting `fine_tune_after_block=-1`) instead of training a linear classifier.

**Comment by chentingpc** [comment]
close the issue for now due to long inactivity.
